OggVorbisEncoder/Lookups/ResidueLookup.cs

```diff
@@ -75,11 +96,11 @@ public ResidueLookup(Residue residue, CodeBook[] fullBooks)
             }
 
             return used
-                ? Res1Forward(buffer, work, 1, partword)
+                ? InnerForward(buffer, new[] { work }, 1, partword)
                 : 0;
         }
 
-        private int Res1Forward(EncodeBuffer buffer, int[] work, int channels, int[][] partword)
+        private int InnerForward(EncodeBuffer buffer, int[][] work, int channels, int[][] partword)
```

Comment:
Converted function to use [][] to mimic how libvorbis functions.
Was problematic when attempting to support other codecs